test/integration/mediaType.spec.ts

```diff
@@ -59,12 +59,14 @@ describe("Integration tests for MediaTypes", () => {
     });
   });
 
-  // https://github.com/Azure/autorest.typescript/issues/741
-  describe.skip("#contentTypeWithEncoding", () => {
+  describe("#contentTypeWithEncoding", () => {
     it("works with text/plain", async () => {
-      const response = await client.contentTypeWithEncoding(
-        "text/plain; encoding=UTF-8"
-      );
+      client = new MediaTypesClient();
+      const response = await client.contentTypeWithEncoding("test", {
+        requestOptions: {
+          customHeaders: { "content-type": "text/plain; encoding=UTF-8" }
+        }
+      });
 
```

Comment:
I think we should also check for the response value.
```suggestion

           expect(response.body).to.equal("Nice job sending content type with encoding");
```

Reply:
I might have messed up the spacing in my suggested code.